x/lockup/types/msgs_test.go

```diff
@@ -95,10 +95,224 @@ func TestMsgLockTokens(t *testing.T) {
 	}
 }
 
-// TODO: Complete table driven tests for the remaining messages
+func TestMsgBeginUnlockingAll(t *testing.T) {
+	appParams.SetAddressPrefixes()
+	pk1 := ed25519.GenPrivKey().PubKey()
+	addr1 := sdk.AccAddress(pk1.Address()).String()
+	invalidAddr := sdk.AccAddress("invalid")
```

Comment:
Seems like we need to generate these three vars for every test. Maybe we can create a function to do this to reduce boilerplate?

Reply:
Yes thanks for the suggestion. I created a function to gen valid & invalid addresses for all tests